tests/integration/core/migrations/test_0056_parse_log_entries.py

```diff
@@ -0,0 +1,222 @@
+#  Copyright 2025 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from datetime import datetime
+from importlib import import_module
+from typing import Type
+from unittest.mock import MagicMock, patch
+
+import pytest
+from django.core.management import call_command
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+from django.db.models import Model
+from django.utils.timezone import make_aware
+
+from aap_eda.core import models
+
+TEST_DATA = [
+    {
+        "msg": "2025-01-17 18:39:49,191 Starting Container",
+        "dt": make_aware(datetime(2025, 1, 17, 18, 39, 49, 191000)),
+        "log": "Starting Container",
+    },
+    {
+        "msg": "** 2025-01-17 18:39:55.222773 [debug] ****",
+        "dt": make_aware(datetime(2025, 1, 17, 18, 39, 55, 222773)),
+        "log": "[debug] ****",
+    },
+    {
+        "msg": "2025-01-17 18:43:38 638 [main] DEBUG org.drools.ansible",
+        "dt": make_aware(datetime(2025, 1, 17, 18, 43, 38, 638000)),
+        "log": "[main] DEBUG org.drools.ansible",
+    },
+]
+
+
+def get_historical_model(
+    app_label: str, model_name: str, migration_name: str
+) -> Type[Model]:
+    """
+    Retrieves the historical model as it existed at the specified migration.
+    """
+    executor = MigrationExecutor(connection)
+
+    # Load the migration state at the specified migration
+    state = executor.loader.project_state((app_label, migration_name))
+
+    return state.apps.get_model(app_label, model_name)
+
+
+@pytest.fixture
+def dummy_rulebook_process(default_organization):
```

Comment:
Do we need a new fixture? 
Can not we reuse what we have in https://github.com/ansible/eda-server/blob/main/tests/integration/conftest.py 
For example https://github.com/ansible/eda-server/blob/main/tests/integration/conftest.py#L409 ?